index.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable */
```

Comment:
- Enable your linters and fix issues.

Reply:
@car I have an issue with h ESLint, but I didn't know how to get rid of them. Can you give me some clues, please
![coach](https://user-images.githubusercontent.com/53347966/135517230-b5e8999c-5677-4e7b-a5c2-e218738d4cf7.JPG)
?